boards/common/atxmega/include/periph_conf_atxmega_common.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_atxmega
+ * @{
+ *
+ * @file
+ * @brief       Common configuration of MCU periphery for ATxmega boards
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_ATXMEGA_COMMON_H
+#define PERIPH_CONF_ATXMEGA_COMMON_H
+
+#include "periph_cpu.h"
+#include "macros/units.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Clock configuration
+ * @{
+ */
+#ifndef CLOCK_CORECLOCK
+#define CLOCK_CORECLOCK     MHZ(32)
+#endif /* CLOCK_CORECLOCK */
+/** @} */
+
+/**
+ * @name    UART configuration
+ *
+ * The UART devices have fixed pin mappings, so all we need to do, is to specify
+ * which devices we would like to use and their corresponding RX interrupts. See
+ * the reference manual for the fixed pin mapping.
+ *
+ * @{
+ */
+
+#ifndef UART_NUMOF
+    #define UART_NUMOF      (0U)
```

Comment:
Style: in general, we don't indent preprocessor statements, so for consistency (even with the rest of this file):

```suggestion
#define UART_NUMOF      (0U)
```
There are other parts affected.